admin_manual/maintenance/update.rst

```diff
@@ -47,6 +47,8 @@ The Updater app performs these operations:
   version
 * Copies your old ``config.php`` to the new ``config/`` directory
 
+.. note:: The backups, created by the updater app, can use up large amount of space if kept over many updates. We recommend that you keep only the ones that you find crucial to your instance and remove the obsolete ones to save storage space.
```

Comment:
Suggestion:
Backups created by the updater app can use large amount of space if kept over many updates. It is recommended to keep only those backups that are crucial to your instance. All others should be removed to save storage space.

Reply:
can you highlight the parts that you changed?

Reply:
You can copy/paste the whole sentence after suggestion... 😄

Reply:
So I should blindly trust you ? :)

Reply:
You should never trust Austrians nor Australians 😄

Reply:
Hey, speak for yourself! Australians are always trustworthy! :-)